nemo/collections/nlp/metrics/classification_report.py

```diff
@@ -97,13 +97,13 @@ def get_precision_recall_f1(
         recall = torch.where(tp + fn != zeros, tp / (tp + fn) * 100, zeros)
         f1 = torch.where(precision + recall != zeros, 2 * precision * recall / (precision + recall), zeros)
 
-        report = '\n{:20s}   {:10s}   {:10s}   {:10s}   {:10s}'.format('label', 'precision', 'recall', 'f1', 'support')
+        report = '\n{:50s}   {:10s}   {:10s}   {:10s}   {:10s}'.format('label', 'precision', 'recall', 'f1', 'support')
```

Comment:
I'll fix this in a separate PR to determine column width based on the max label length

Reply:
Currently for my case it creates unreadable table. So, let's have this with 50 for now and then you can improve it for generic case.